contents/data-display/avatars/avatar-with-text/index.tsx

```diff
@@ -1,19 +1,21 @@
-import { Avatar, Box, Center, Text } from "@yamada-ui/react"
+import { Avatar, HStack, Text, VStack } from "@yamada-ui/react"
 import type { FC } from "react"
 
 const AvatarWithText: FC = () => {
   return (
-    <Center gap="sm">
+    <HStack>
       <Avatar
-        size="md"
         src="https://avatars.githubusercontent.com/u/109452865?v=4"
         name="Taku"
       />
-      <Box>
-        <Text>Taku</Text>
-        <Text>@taku10101</Text>
-      </Box>
-    </Center>
+
```

Comment:
This line break might not be necessary !

Reply:
Fixed.